unity_buy_android/unity_buy_plugin/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.0"
+
+    defaultConfig {
+        minSdkVersion 16
```

Comment:
Min version is 16? You sure you want to support this old OS:
https://developer.android.com/about/dashboards/index.html

I believe you can easliy bump it up to 19. But it's up to you